avocado/core/test.py

```diff
@@ -691,6 +694,30 @@ def fetch_asset(self, name, asset_hash=None, algorithm='sha1',
         return asset.Asset(name, asset_hash, algorithm, locations,
                            self.cache_dirs, expire).fetch()
 
+    @staticmethod
+    def skipIf(condition, reason):
```

Comment:
My concern here is placement. In the `unittest` module, the `skipIf` decorator is part of the main `unittest` namespace:

    from unittest import skipIf

Also, structurally, the decorator doesn't seem to belong to the `Test` class. I'd suggest that those are moved to some library, and make it also directly available under `avocado.skipIf`.